java/dataset/src/main/java/org/apache/arrow/dataset/jni/DirectReservationListener.java

```diff
@@ -40,10 +38,13 @@ private DirectReservationListener() {
       methodUnreserve = this.getDeclaredMethodBaseOnJDKVersion(classBits, "unreserveMemory");
       methodUnreserve.setAccessible(true);
     } catch (Exception e) {
-      final RuntimeException failure = new RuntimeException(
-          "Failed to initialize DirectReservationListener. When starting Java you must include " +
-              "`--add-opens=java.base/java.nio=org.apache.arrow.dataset,org.apache.arrow.memory.core,ALL-UNNAMED` " +
-              "(See https://arrow.apache.org/docs/java/install.html)", e);
+      final RuntimeException failure =
+          new RuntimeException(
+              "Failed to initialize DirectReservationListener. When starting Java you must include "
+                  + "`--add-opens=java.base/java.nio=org.apache.arrow.dataset,"
+                  + "org.apache.arrow.memory.core,ALL-UNNAMED` "
+                  + "(See https://arrow.apache.org/docs/java/install.html)",
+              e);
```

Comment:
Applied spotless format, but the final result was one line was 121 characters and the checkstyle caught it and I did a manual change here. That's okay?

Reply:
Probably I should do that too to keep it consistent.

Reply:
Updated.